bs-react-native-next/src/apis/NetInfo.md

```diff
@@ -0,0 +1,66 @@
+---
+id: apis/NetInfo
+title: NetInfo
+wip: true
+---
+
+```reason
+type info;
+
+[@bs.obj]
+external info:
+  (
+    ~effectiveType: [@bs.string] [
+                      | [@bs.as "2g"] `Net2G
+                      | [@bs.as "3g"] `Net3G
+                      | [@bs.as "4g"] `Net4G
+                      | `unknown
+                    ]
+                      =?,
+    ~_type: [@bs.string] [
+              | `bluetooth
+              | `cellular
+              | `ethernet
+              | `unknown
+              | `wifi
+              | `wimax
+            ]
+              =?,
+    unit
+  ) =>
+  info =
+  "";
+
+[@bs.get] external effectiveType: info => string = "";
+[@bs.get] external _type: info => string = "type";
```

Comment:
Also here: Use `type_` instead.

Reply:
type_ won’t work automagically with bs.obj like _type does.

So I would keep it as it is.

Reply:
> Also here: Use `type_` instead.

I transitioned from `type_` to `_type`, etc. a while ago, but forgot why. According to [BuckleScript documentation](https://bucklescript.github.io/bucklescript/Manual.html#_object_label_translation_convention) 

> Else if the rest happens to be a keyword, drop the first '_'

So, for keywords I believe it's better to use `_type` everywhere.